packages/zwave-js/src/lib/commandclass/UserCodeCC.ts

```diff
@@ -755,8 +765,14 @@ export class UserCodeCC extends CommandClass {
 			return;
 		}
 
+		for (let userId = 1; userId <= supportedUsers; userId++) {
+			setUserCodeMetadata.call(this, userId);
+		}
+
 		// Synchronize user codes and settings
-		await this.refreshValues();
+		if (this.driver.options.interview?.queryAllUserCodes) {
```

Comment:
```suggestion
		if (this.driver.options.interview.queryAllUserCodes) {
```
After applying the default options, this is always defined.

Reply:
Same goes for the other locations.

Reply:
oh duh, i just realized you're doing a deep merge in the Driver constructor using the default options. Got it. Let me roll back these

Reply:
fixed, via git commit --amend + force push

Reply:
> git commit --amend + force push

You generally don't need to do that here - just makes following the changes in a PR harder. I squash merge every PR for a clean history.